src/components/Tooltip.js

```diff
@@ -3,12 +3,15 @@ import PropTypes from 'prop-types';
 
 import { UncontrolledTooltip } from 'reactstrap';
 
-const Tooltip = ({ text, children, ...props }) => {
+const Tooltip = ({ text, children, wrapperTag, tooltipProps, ...props }) => {
```

Comment:
No current usage of `Tooltip` sets any props besides `text`, so this is a non-breaking change.